custom_model_runner/datarobot_drum/drum/perf_testing.py

```diff
@@ -698,6 +699,9 @@ def check_prediction_side_effects(self):
 
             preds_full_subset = preds_full.iloc[data_subset.index]
 
+            # Validate that the predictions are of the type and shape the user specified in the schema
+            self._schema_validator.validate_outputs(preds_sample)
```

Comment:
what if this is none?

Reply:
It will always be a dataframe, even if empty.

Reply:
not according to your init

Reply:
the validator not the data.  right.